tests/Extensions/Hosting.Services.Web.UnitTests/WebEndpointInfoTests.cs

```diff
@@ -22,7 +22,7 @@ public void CreateHttp_SetsProperties()
 			Assert.IsNull(info.SettingForCertificateCommonName);
```

Comment:
I'm OK with this change – Semmle is obviously reporting a non-issue here – but in reality would we or library consumers ever use the `CreateHttp()` method? HTTP feels very outdated to me know, so perhaps the real fix here is to remove the offending method rather than suppressing the issue in the test.

Reply:
It's still needed, but we could make it obsolete

Reply:
Done here https://github.com/microsoft/Omex/pull/297